ozaria/site/components/cutscene/PageCutscene/index.vue

```diff
@@ -61,28 +43,21 @@ module.exports = Vue.extend({
 
 <template>
   <layout-chrome>
-    <layout-center-content>
-      <div id="cutscene-player"
-      :style="{ width: width+'px', height: height+'px' }" v-if="vimeoId">
-      <!-- Currently this video is a hard coded example, that will be fetched from Cutscene collection -->
-        <base-video
-          :vimeoId="vimeoId"
-          :width="width"
-          :height="height"
+    <base-video
+      :vimeoId="vimeoId"
+      id="cutscene-player"
 
-          v-on:completed="onCompleted"
-        />
-      </div>
-    </layout-center-content>
+      v-on:completed="onCompleted"
+    />
   </layout-chrome>
-
 </template>
 
 <style lang="sass">
+  @import "ozaria/site/styles/common/variables.sass"
 
-#cutscene-player
-  margin-left: auto
-  margin-right: auto
+  #cutscene-player
+    width: calc(100vw - #{$chromeRightPadding + $chromeLeftPadding})
+    height: calc(100vh - #{$chromeTopPadding + $chromeBottomPadding})
```

Comment:
How do we handle vertical / horizontal centering?

Reply:
The centering is handled by vimeo. Provided the iframe is expanded to max size, vimeo uses a flex box to center the video.